ONSdigital/ras-frontstage
# update design system version

# What and why?
Updating design system version to 62.2.4.

# How to test?
Deploy to environment, run acceptances tests, and navigate around frontstage to make sure everything looks okay.

# Jira
[Card](https://jira.ons.gov.uk/browse/RAS-816)
